openapi/api.yml

```diff
@@ -544,6 +585,45 @@ paths:
           description: Not Found
       security:
         - Bearer: []
+  '/transactions/{id}/actions/resume3ds2':
```

Comment:
Tnx, I'll use this api to complete test on pending [PR resume test](https://github.com/pagopa/io-pay/pull/38) . what do you think about it? Does this make sense?